cip-drep-metadata/README.md

```diff
@@ -0,0 +1,231 @@
+
+This CIP has not included these features, the decision not to include these features was made in order to simplify this CIP so that it became suitable for the minimum viable level of tooling, with the expectation that further CIPs will build on it. 
+
+## Specification
+CIP-1694 specifies that metadata anchors are optional for DRep registration and updates. This CIP covers metadata for the aforementioned transaction types, but it does not cover metadata for voting transactions. 
+
+### Markdown Text Styling
+Unlike [CIP-108](https://github.com/Ryun1/CIPs/blob/governance-metadata-actions/CIP-0108/README.md#markdown-text-styling) this standard seeks to simplify the job of tooling providers by not supporting text formatting such as markdown. 
+
+### Integrity
+Metadata is said to have integrity when the hash of this metadata in the metadata anchor on-chain is the same as a hash made of the metadata now (given that the same algorithm is used). Tooling SHOULD check for metadata integrity, with the hashing algorithm determined by rules outlined in [CIP-100](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100). If tooling detects that the metadata does not have integrity it MUST NOT display the metadata, and it MUST display a warning describing why it is not showing the metadata.
+
+### Formatting
+If DRep Metadata is not formatted in the way described by this CIP then tooling SHOULD either:
+1. Deal with it as described in CIP-100 (by rendering the raw unformatted document)
+2. Not display the any of the metadata
+In either case the tooling SHOULD provide a warning that this has happened and link to the url described in the metadata anchor of the DRep registration.
+
+### Missing
+If the DRep metadata cannot be found at the address specified in the anchor then the tooling SHOULD provide a warning. There is no need to link to the url in the metadata anchor. 
+
+### Witnesses
+DRep Metadata will not follow the CIP-100 specification related to signing the metadata, instead they are attached to a DRep registration. The need for an `authors` field will also be discarded in favour of including usernames inside of the `body` field. 
+
+### Extended Vocabulary
+Like CIP-108, this CIP also extends the potential vocabulary of CIP-100's `body` property. For all of these properties tooling providers MUST be aware that they are responsible for what they display to their users and that these fields could be used for illegal, unsavoury, or innapropriate purposes or language. Therefore such tools SHOULD have a terms of service which they enforce to moderate what they show to their users. The following are a list of properties tooling should expect to encounter:
```

Comment:
this would be defined by their TOS, but I was thinking of people using the metadata as a vector to be insulting or defamatory to other people or community members.

Reply:
I would be very worried about encoding centralised moderation or censorship into the CIP itself.  I believe the place for these things is in the tool documentation and, as you say, the user terms of service for those governance tools.  It is not part of the metadata framework itself to mandate censorship, and in fact that framework _itself_ will work fine without that censorship.  If there are reasons why such censorship is beneficial, or considered practically inevitable, then they should be documented in the practical materials for developing and using the tools, not the CIP: otherwise we invite a subjectivity into the CIP process that has long been avoided.

Reply:
@rphair ok.

Reply:
@Thomas-Upfield thanks for your attention but there are a couple remaining issues with the relevant change in https://github.com/cardano-foundation/CIPs/pull/788/commits/c0fb5b6d6135c88c40d516d11fdabb95f1e71750:
- A short trailing paragraph has been, I assume inadvertently, included in the **Note**: introducing the list of properties: there would need to be a paragraph break before the final sentence.
- The issue is not resolved by changing the typography from a paragraph to a "Note" since it still makes the insupportable claims that the CIP can "enforce" censoring TOS's on tooling providers and also in turn mandate own "enforcement" of those terms on users.

There is a similar problem with the "intent" of the `username` field avoid "crass language"... no less a problem without an equally undignified inventory of which language is unacceptable (which itself would be "crass").

I suspect there is a bureaucratic mandate behind these repeated interjections of moderation (censorship) into what should be a straightforward technical document, so in my experience there would need to be a floodlight of visibility on this to decide how to move ahead as a community.

We have a light CIP agenda coming up so, given that this document is still being refined, I've added this one item to our agenda & would hope you can be there to explain the origin of this apparent moderation requirement and hopefully get some consensus about whether & how it might have a place in this (or any) CIP.  cc @Ryun1 @Crypto2099